homeassistant/components/cover/knx.py

```diff
@@ -34,6 +39,9 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_SETPOSITION_ADDRESS): cv.string,
     vol.Optional(CONF_INVERT_POSITION, default=False): cv.boolean,
+    vol.Optional(CONF_GETANGLE_ADDRESS): cv.string,
+    vol.Optional(CONF_SETANGLE_ADDRESS): cv.string,
```

Comment:
use `vol.Include(..., 'angle'): cv...` that make that the user need set this options.